spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,9 +56,14 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
         add_product_to_cart order, product_tax
       end
 
+      after do
+        Spree::Config.set allow_backorders: true
```

Comment:
This is probably doing the right thing, but not very clean. It would be correct to save the previous state so that you can restore it at this point.